src/main/java/streams/cta/io/ImageStream.java

```diff
@@ -53,15 +62,23 @@ public Data readNext() throws Exception {
             return null;
         }
 
```

Comment:
sorry, this comment function is not as I would have awaited it to be (the "good looking" code is below the marked position!)